packages/tradershub/src/flows/FirstSignup/FirstSignupScreens/FirstSignupScreens.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { Dispatch } from 'react';
+import { Modal } from '@/components';
+import { CitizenshipModal } from '../CitizenshipModal';
+
+type TFirstSignupScreens = {
+    setStep: Dispatch<React.SetStateAction<number>>;
+    step: number;
+};
+
+const FirstSignupScreens = ({ step, setStep }: TFirstSignupScreens) => {
```

Comment:
@shahzaib-deriv will rename the previous SignupWizard to RealAccountWizard so don't need to call it FirstSignupScreen, we can call it Signupscreens